### ISSUE TITLE
Update for esbuild 13

### ISSUE THREAD

**Issue by zackgomez** [craftamap/esbuild-plugin-html#1]
Hi love this package, seems to work great under esbuild 13 but I get a peer dependency warning

```
warning " > @craftamap/esbuild-plugin-html@0.2.0" has incorrect peer dependency "esbuild@^0.12.28".
```

It would be great to update this package to be marked as supporting the new version of esbuild

**Comment by craftamap** [comment]
Hey there, thanks for liking the package!

I just published v0.2.1 which should resolve this issue: https://www.npmjs.com/package/@craftamap/esbuild-plugin-html/v/0.2.1

Please report back if it works :)

**Comment by zackgomez** [comment]
I can confirm I no longer get the warning and the plugin still works for me.

Thank you for the lightning fast turnaround time!
